Documentation/ApiOverview/Fal/UsingFal/ExamplesFileFolder.rst

```diff
@@ -19,15 +19,11 @@ Getting a file
 
 A file can be retrieved using its uid:
 
-.. code-block:: php
```

Comment:
If you do this, you **must** use `::`` (double colon) in the paragraph before the code block. Otherwise it will not be detected as such but as a quote instead.

See "shorthand" here: https://docs.typo3.org/m/typo3/docs-how-to-document/master/en-us/WritingReST/Codeblocks.html